HIRS_AttestationCA/src/main/java/hirs/attestationca/persist/entity/userdefined/PolicySettings.java

```diff
@@ -47,38 +48,32 @@ public class PolicySettings extends UserDefinedEntity {
     @Column(nullable = false, columnDefinition = "boolean default false")
     private boolean firmwareValidationEnabled = false;
 
-    @Column(nullable = false, columnDefinition = "boolean default false")
-    private boolean utcValidationEnabled = false;
-
     @Column(nullable = false, columnDefinition = "boolean default false")
     private boolean expiredCertificateValidationEnabled = false;
 
-    @Column(nullable = false, columnDefinition = "boolean default false")
-    private boolean replaceEC = false;
-
     @Column(nullable = false, columnDefinition = "boolean default true")
-    private boolean issueAttestationCertificate = true;
+    private boolean issueAttestationCertificateEnabled = true;
 
     @Column(nullable = false, columnDefinition = "boolean default true")
-    private boolean issueDevIdCertificate = true;
+    private boolean issueDevIdCertificateEnabled = true;
 
     @Column(nullable = false)
-    private String validityDays = TEN_YEARS;
+    private Long validityDays = TEN_YEARS_IN_DAYS;
 
     @Column(nullable = false)
-    private String devIdValidityDays = TEN_YEARS;
+    private Long devIdValidityDays = TEN_YEARS_IN_DAYS;
 
     @Column(nullable = false)
-    private String reissueThreshold = YEAR;
+    private Long reissueThreshold = A_YEAR_IN_DAYS;
 
     @Column(nullable = false)
-    private String devIdReissueThreshold = YEAR;
+    private Long devIdReissueThreshold = A_YEAR_IN_DAYS;
 
     @Column(nullable = false, columnDefinition = "boolean default false")
-    private boolean generateOnExpiration = false;
+    private boolean generateAttestationCertificateOnExpiration = false;
```

Comment:
I'm pointing out this line as a matter of future-proofing/concern.  While I agree that the name change makes this variable's purpose a little more clear in the code, the new name is rather different from the corresponding backend column.  If this causes confusion in the future as to what column this variable corresponds to, then we've created a problem for ourselves.  (The same could be said for issueAttestationCertificate --> issueAttestationCertificateEnabled above, but that relationship remains fairly obvious even after the name change.)